forest/src/cli/chain_cmd.rs

```diff
@@ -49,6 +68,46 @@ impl ChainCommands {
                 let cid: Cid = cid.parse().unwrap();
                 print_rpc_res_pretty(chain_get_block((CidJson(cid),)).await);
             }
+            Self::Export {
+                tipset,
+                recent_stateroots,
+                skip_old_messages,
+                output_path,
+            } => {
+                let chain_head = match chain_head().await {
+                    Ok(head) => head.0,
+                    Err(_) => return cli_error_and_die("Could not get network head", 1),
+                };
+
+                let epoch = tipset.unwrap_or(chain_head.epoch());
+
+                let output_path = match output_path {
+                    Some(path) => path.to_owned(),
+                    None => {
+                        let now = OffsetDateTime::now_utc();
+                        format!(
+                            "forest_snapshot_{}-{}-{}_height_{}.car",
+                            now.year(),
+                            now.month(),
+                            now.day(),
+                            epoch,
+                        )
+                    }
+                };
+
+                let params = (
+                    epoch,
+                    *recent_stateroots,
+                    *skip_old_messages,
+                    output_path.clone(),
```

Comment:
why do we need to clone it? It's not used anymore, no?

Reply:
Fixed.